demo/src/main/java/foundation/algorand/demo/AnswerActivity.kt

```diff
@@ -123,6 +128,13 @@ class AnswerActivity : AppCompatActivity() {
             "(Android ${Build.VERSION.RELEASE}; ${Build.MODEL}; ${Build.BRAND})"
     private var signature: ByteArray? = null
 
+    // Datachannel Provider/Handler
+    val uuidGenerator = Generators.timeBasedEpochRandomGenerator()
+
+    // Must be unique to this provider
+    val providerId = uuidGenerator.generate().toString()
+    private val provider = AVMProvider(providerId)
```

Comment:
The AVMProvider is being imported from `demo` and we are creating the class in the demo package. 

Can't we just import the actual package and use that?

Reply:
We could offer a base interface with empty stubs or go with a more advanced strategy. The wallets would need to handle the signing operations and anything extra like Bio-metrics. We only are dependent on the message shapes/encoding at this point